From 15134960fcb2674b5af7cdfb52342cfe304e8acc Mon Sep 17 00:00:00 2001
From: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
Date: Mon, 16 Apr 2018 13:20:32 -0500
Subject: [PATCH] FROMGIT: ASoC: SOF: enable DEBUG by default

BugLink: https://bugs.launchpad.net/bugs/1826181

better to use dynamic debug for products but it's really convenient

Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
(cherry picked from commit c99938cbd430b48a9e4b45ae0d450eeab59031d9
git://github.com/thesofproject/linux.git)
Signed-off-by: Hui Wang <hui.wang@canonical.com>
---
 sound/soc/sof/Makefile        | 2 ++
 sound/soc/sof/intel/Makefile  | 2 ++
 sound/soc/sof/xtensa/Makefile | 2 ++
 3 files changed, 6 insertions(+)

diff --git a/sound/soc/sof/Makefile b/sound/soc/sof/Makefile
index 8f14c9d2950b..33c3ded2b7e2 100644
--- a/sound/soc/sof/Makefile
+++ b/sound/soc/sof/Makefile
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
 
+ccflags-y += -DDEBUG
+
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
 		control.o trace.o utils.o
 
diff --git a/sound/soc/sof/intel/Makefile b/sound/soc/sof/intel/Makefile
index 84a65c80f62a..03b22924d0c4 100644
--- a/sound/soc/sof/intel/Makefile
+++ b/sound/soc/sof/intel/Makefile
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
 
+ccflags-y += -DDEBUG
+
 snd-sof-intel-byt-objs := byt.o
 snd-sof-intel-hsw-objs := hsw.o
 snd-sof-intel-bdw-objs := bdw.o
diff --git a/sound/soc/sof/xtensa/Makefile b/sound/soc/sof/xtensa/Makefile
index cc89c7472a38..34b27be0f7be 100644
--- a/sound/soc/sof/xtensa/Makefile
+++ b/sound/soc/sof/xtensa/Makefile
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
 
+ccflags-y += -DDEBUG
+
 snd-sof-xtensa-dsp-objs := core.o
 
 obj-$(CONFIG_SND_SOC_SOF_XTENSA) += snd-sof-xtensa-dsp.o
-- 
2.17.1

